File: src/java/Median2sortarray.java
import java.util.List;
import java.util.Collections;
import java.util.Arrays;
import java.util.ArrayList;

public class Median2sortarray {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
	List<Integer> l = new ArrayList<Integer>();
	for (int i =0; i<nums1.length; i++)
	    l.add(nums1[i]);
	for (int i=0; i<nums2.length; i++)
	    l.add(nums2[i]);
        Collections.sort(l);
        if (l.size() % 2 == 1)
            return l.get(l.size()/2);
        else
            return ((int)l.get(l.size()/2) + (int)l.get(l.size()/2-1) ) / 2.0;        
    }

    public static void main(String argv[]){
	int[] a = {1,2,3,4,5};
	int[] b = {6,7,8,9,10};
	Median2sortarray m = new Median2sortarray();
	for (int i = 0; i< a.length; i++)
	    System.out.println(a[i]);
        for (int i = 0; i< b.length; i++)
            System.out.println(b[i]);
	System.out.println(m.findMedianSortedArrays(a, b));
    }
}

File: src/python/median2sortarray.py

class Solution(object):
    def findMedianSortedArrays(self, nums1, nums2):
        """
        :type nums1: List[int]
        :type nums2: List[int]
        :rtype: float
        """
	nums1.extend(nums2)
	nums1.sort()
	l = len(nums1)
	if l%2==0:
	    return (nums1[l/2] + nums1[l/2-1])/2.0
	else:
	    return (nums1[l/2])	
		

if __name__ == '__main__':
    nums1 = [1,3]
    nums2 = [2,4]
    s = Solution()
    print s.findMedianSortedArrays(nums1, nums2)
